themes/bootstrap5/templates/RecordDriver/EDS/result-list.phtml

```diff
@@ -113,6 +116,39 @@
       <?php if ($identifierLinksHtml): ?>
         <div><?=$identifierLinksHtml?></div>
       <?php endif; ?>
+      <?php /* Format / Label / Clear Availability Display */ ?>
+      <?php
+        $openUrl = $this->openUrl($this->driver, 'results');
```

Comment:
I think the OpenURL logic was intentionally excluded from EDS because the link resolver link tends to come through the EDS API directly, which could lead to weird redundancy... but maybe I'm misremembering. Just curious if you've already looked into this detail.